test/parallel/test-http-agent-keepalive.js

```diff
@@ -102,7 +102,7 @@ function remoteClose() {
 }
 
 function remoteError() {
-  // remove server will destroy ths socket
+  // remove server will destroy the socket
```

Comment:
I think "remove" here is supposed to be "remote"?

Reply:
Ahhh yeah sure, thanks!